dietza/whats-cookin
# Feature/document object model

# Pull Request Template:

#### What's this PR do?
This PR created ingredientRepository, recipeRepository, scripts.JS, but it mostly adds features to the DOM. 

#### Where should the reviewer start?
The reviewer should start at scripts.js and open index.html, then open devtools and familiarize themselves with the sidebar changes and navigation updates. 

#### How should this be manually tested?
The user should manually test index.html in the DOM. The user should also manually test the classes to verify that they are being instantiated on load. 

#### Any background context you want to provide?
n/a

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate):
n/a

#### Questions
